docs/specification/draft/basic/transports.md

```diff
@@ -48,38 +48,209 @@ sequenceDiagram
     deactivate Server Process
 ```
 
-## HTTP with SSE
+## Streamable HTTP
 
-In the **SSE** transport, the server operates as an independent process that can handle
-multiple client connections.
+{{< callout type="info" >}} This replaces the [HTTP+SSE
+transport]({{< ref "/specification/2024-11-05/basic/transports#http-with-sse" >}}) from
+protocol version 2024-11-05. See the [backwards compatibility](#backwards-compatibility)
+guide below. {{< /callout >}}
 
-The server **MUST** provide two endpoints:
+In the **Streamable HTTP** transport, the server operates as an independent process that
+can handle multiple client connections. This transport uses standard HTTP with optional
+[Server-Sent Events](https://en.wikipedia.org/wiki/Server-sent_events) (SSE) for
+streaming multiple server messages. This permits "plain HTTP" MCP servers, as well as
+more feature-rich servers supporting streaming and server-to-client notifications and
+requests.
 
-1. An SSE endpoint, for clients to establish a connection and receive messages from the
-   server
-2. A regular HTTP POST endpoint for clients to send messages to the server
+The server **MUST** provide a single HTTP endpoint path (hereafter referred to as the
+**MCP endpoint**) that supports both POST and GET methods. For example, this could be a
+URL like `https://example.com/mcp`.
```

Comment:
Would it be advantageous to make this a [`.well-known`](https://en.wikipedia.org/wiki/Well-known_URI) path?

If so what would the process be to get it added?